tcms/testruns/tests/test_views.py

```diff
@@ -306,7 +306,7 @@ def test_404_if_run_not_exist(self):
         self.assert404(response)
 
     def assert_cc(self, response, expected_cc):
-        self.assertEqual(len(expected_cc), self.test_run.cc.count())
+        self.assertEqual(len(expected_cc), self.test_run.cc.count())  # pylint: disable=no-member
```

Comment:
Revert this change. That sounds like an issue with pylint-django and it should be fixed there. The rest is fine.

Reply:
Done. IMO it is just not able to infer the type of `test_run` correctly